index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <script src="https://kit.fontawesome.com/e520ead38a.js"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <a href="#" class="logo">MASSAM</a>
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <main>
+        <section class="headline-section">
+            <h1>I'm Massam.<br>Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website 
+                Look through some of my work and experience! If you like what you see and have 
+                a project you need coded, don’t hesitate to contact me.
+            </p>
+            <label for="media-icons">LET'S CONNECT</label>
```

Comment:
- [OPTIONAL] This  `label` element has a `for` attr, but no element with `id` or `name` that matches the value of `for` exist, usually `label` tags are used with `input` tags. I would highly suggest that you change this `label` to a `button`, what do you think?

Reply:
Basically it is a heading only so rather than chnging it to button I would prefer to change it to an anchor tag or just leave it as a label and remove the for attribute from it.